pkgs/top-level/all-packages.nix

```diff
@@ -9062,6 +9062,10 @@ in
   # Should eventually be moved inside Python interpreters.
   python-setup-hook = callPackage ../development/interpreters/python/setup-hook.nix { };
 
+  pyjson5 = callPackage ../development/python-modules/pyjson5 {
```

Comment:
Things from pkgs/development/python-modules can not go to all-packages.nix, put those in python-packages.nix instead